.github/workflows/windows.yml

```diff
@@ -362,20 +362,36 @@ jobs:
           name: ${{ needs.openvino_download.outputs.ov_artifact_name }}
           path: ${{ env.INSTALL_DIR }}
           merge-multiple: true
-
-      - name: Install OpenVINO Python wheel from pre-built artifacts
-        run: |
-          python3 -m pip install openvino==${{ needs.openvino_download.outputs.ov_version }} ${{ needs.openvino_download.outputs.ov_wheel_source }}
-        working-directory: ${{ env.INSTALL_DIR }}
-
+        
+      - name: Configure poetry
+        uses: ./openvino_tokenizers/.github/actions/install_poetry
+      
+      - name: Find OpenVINO wheel
+        uses: ./openvino_tokenizers/.github/actions/find_wheel
+        id: ov_wheel
+        with:
+          wheels_dir: '${{ env.INSTALL_DIR }}\\wheels'
+          package_name: 'openvino'
+         
+      - name: Install OpenVINO wheel
+        if: ${{ github.actor != 'dependabot[bot]' }}
+        run: poetry add ${{ steps.ov_wheel.outputs.wheel_path }}
```

Comment:
can we use `poetry run pip install openvino --find-links ${{ env.INSTALL_DIR }}\\wheels` ?